src/components/editor/property/InputListLOC.jsx

```diff
@@ -6,27 +6,25 @@ import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import shortid from 'shortid'
 import { changeSelections } from 'actions/index'
-import { getDisplayValidations } from 'selectors/resourceSelectors'
-import { booleanPropertyFromTemplate, defaultValuesFromPropertyTemplate } from 'Utilities'
+import { itemsForProperty, getDisplayValidations, getPropertyTemplate } from 'selectors/resourceSelectors'
+import { booleanPropertyFromTemplate, defaultValuesFromPropertyTemplate, getLookupConfigItems } from 'Utilities'
 
+// propertyTemplate of type 'lookup' does live QA lookup via API
+//  based on URI in propertyTemplate.valueConstraint.useValuesFrom,
+//  and the lookupConfig for the URI has component value of 'list'
 class InputListLOC extends Component {
   constructor(props) {
     super(props)
     this.state = {
       isLoading: false,
       options: [], // The suggestions returned from QA
-      defaults: defaultValuesFromPropertyTemplate(this.props.propertyTemplate),
-      error: null,
     }
   }
 
   componentDidMount() {
     this._isMounted = true
-    if (this.state.defaults.length > 0) {
-      this.selectionChanged(this.state.defaults)
-    } else {
-      // Property templates do not require defaults but we like to know when this happens
```

Comment:
We decided to ditch this kind of message in another spot, so I ditched it here also.  I suspect this is from earlier efforts when we were less confident of our handling of default values.